web/olga/charts/templates/charts/base.html

```diff
@@ -65,7 +65,7 @@
                 <div class="grid-container grid-manual">
                     <div class="row">
                         <div id="footer-text" class="col col-12 sm-col-12 md-col-6 text-right">
-                            <p class="copyright">by <a href="raccoongang.com">Raccoon Gang</a>, 2018</p>
+                            <p class="copyright">by <a href="https://raccoongang.com/">Raccoon Gang</a>, 2018</p>
```

Comment:
Why do we need this change?

Reply:
It's fixing for this [bug](https://youtrack.raccoongang.com/issue/OLGA-50).